sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_digitaltwins_client.py

```diff
@@ -563,11 +564,11 @@ def list_event_routes(self, **kwargs):
 
     @distributed_trace
     def upsert_event_route(self, event_route_id, event_route, **kwargs):
-        # type: (str, "models.EventRoute", **Any) -> None
+        # type: (str, "DigitalTwinsEventRoute", **Any) -> None
         """Create or update an event route.
 
         :param str event_route_id: The Id of the event route to create or update.
-        :param ~azure.digitaltwins.models.EventRoute event_route: The event route data.
+        :param DigitalTwinsEventRoute event_route: The event route data.
```

Comment:
In docstrings, you should use the fully qualified name to enable hyper-linking:
`:param ~azure.digitaltwins.DigitalTwinsEventRoute:`

Reply:
Fixed